sdk/tests/wallet/core.rs

```diff
@@ -97,13 +97,14 @@ async fn changed_bip_path() -> Result<()> {
         .finish()
         .await;
 
-    // Building the wallet with another coin type needs to return an error, because a different coin type was used in
-    // the existing account
-    let mismatch_err: Result<Wallet> = Err(Error::BipPathMismatch {
+    let _mismatch_err: Result<()> = Err(Error::BipPathMismatch {
```

Comment:
any reason for the underscore?

Reply:
clippy warning (bc the binding is not really used, just matched)

Reply:
That sucks, sounds like a clippy issue tbh, if ever you want to report it

Reply:
it definitely struck we as a bit weird

Reply:
I don't think they are tbh, it's clearly being used 🤷🏻‍♂️